R/assets.R

```diff
@@ -42,12 +53,15 @@ earth_asset <- function(key, lon, lat, start_date, end_date = Sys.Date()){
                   "lon=", lon, "&",
                   "lat=", lat, "&",
                   "begin=", start_date, "&",
-                  if(difdate){paste0("end=",end_date,
-                                     "&api_key=",key)
-                    }else{paste0("&api_key=",key)})
-  s <- jsonlite::fromJSON(query)
+                  if (difdate) {
+                    paste0("end=", end_date, "&api_key=", key)
+                  } else {
+                    paste0("&api_key=",key)
+                  })
+
+  s <- fromJSON(query)
 
-  if("error" %in% names(s)){
+  if ("error" %in% names(s)) {
```

Comment:
if I may suggest, I would later add support for `lintr`. It can detect some semantic errors through static analysis of code and check for code formatting rules. One of the default rules is actually space after control statements such as `if` and space before and after curly braces `{`, `}`.